builder/vsphere/common/step_import_to_content_library.go

```diff
@@ -46,6 +54,8 @@ type ContentLibraryDestinationConfig struct {
 	Datastore string `mapstructure:"datastore"`
 	// If set to true, the VM will be destroyed after deploying the template to the Content Library. Defaults to `false`.
 	Destroy bool `mapstructure:"destroy"`
+	// TODO
```

Comment:
commenting here just in case we forget about this later on

Reply:
I'm going to change this. I committed the changes so I could share the artifacts with the issue's reporter. I wasn't sure if this implementation would solve their case but it did 🎉

Reply:
So I'm still working with the docs cuz I need to decide what's going to be